lib/rules/declaration-property-value-no-unknown/index.mjs

```diff
@@ -69,6 +69,9 @@ const rule = (primary, secondaryOptions) => {
 
 		/** @type {SecondaryOptions['propertiesSyntax']} */
 		const propertiesSyntax = {
+			'overflow-x': '| <-non-standard-overflow>',
+			'overflow-y': '| <-non-standard-overflow>',
+			'position-try-options': "<'position-try-fallbacks'>",
```

Comment:
Based on [this discussion](https://github.com/stylelint/stylelint/issues/7964#issuecomment-2351588284), Stylelint shouldn't actively support removed or deprecated CSS features unless they have a broad impact. I think this change shouldn't be made. What do you think?

Reply:
I think it's harmless and it will be discussed more seriously once we will transition to #8100.
i.e. #8100 doesn't plan to support that kind of renames

Reply:
As far as I know, extending already renamed features like this hasn't been done before.
Adding it here would set a precedent.

Since multiple members have expressed opposition to this change, I think we should avoid adding it.
I’m also against it. If needed, users can easily extend it themselves.

Reply:
What I am doing for `overflow-x` and `overflow-y` is similar; these values are non-standard but are or used to be supported by browsers.
i.e. either we accept the PR as a whole on the same principle or we don't ever support anything "non-standard" and close the PR

Reply:
Sorry for the late reply.
I submitted a PR to CSSTree to handle these properties upstream:

- https://github.com/csstree/csstree/pull/324
- https://github.com/csstree/csstree/pull/323

This was based on [Romain's comment](https://github.com/stylelint/stylelint/issues/7964#issuecomment-2351588284):

> (You could open an issue upstream with either csstree or mdn-data to request that they preserve position-try-options as valid.)

Let me know what you think.

Reply:
That's great but since none of us have write access on csstree, it might take a while.
Let's wait and see.